src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useCallback, useEffect, useState } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+const getCenturyRange = (century: number): [number, number] => {
+  const start = (century - 1) * 100 + 1;
+  const end = century * 100;
+
+  return [start, end];
+};
+
+export const PeoplePage = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
you can just set loading to true initially, no need to set it at the beggining of useEffect

Reply:
Indeed!